contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol

```diff
@@ -191,8 +191,8 @@ contract DefaultReserveInterestRateStrategy is IReserveInterestRateStrategy {
   }
 
   /// @inheritdoc IReserveInterestRateStrategy
-  function calculateInterestRates(DataTypes.CalculateInterestRatesParams calldata params)
-    external
+  function calculateInterestRates(DataTypes.CalculateInterestRatesParams memory params)
+    public
```

Comment:
@miguelmtzinf is it needed?

Reply:
Github's diff is broken. These changes are already part of the codebase